src/main/api/2/studio-api-2.yaml

```diff
@@ -4592,6 +4594,131 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /clipboard/copy:
+    post:
+      tags:
+        - clipboard
+      summary: Copy the given items to the target path
+      description: 'Required permission "Write"'
+      operationId: copyItems
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ClipboardRequest'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+                  items:
+                    description: The list of the new items
```

Comment:
How would we have list of new items already if we haven't pasted? Is the copy operation actually the paste from a prior copy? Same goes for cut spec below.

If this is the case, I think even though at a disk/git level copy and cut are the actual ops (i.e. cp, mv), in this context ("clipboard") the mental model is more copy + paste. Isn't `paste` named copy/cut unexpected? Shall we consider having a single `/clipboard/paste` with a `request.type = 'copy' | 'cut'`. Or if we wish to keep them separate, perhaps `/clipboard/paste_from_copy` and  `/clipboard/paste_from_cut`?

If we are going to keep the clipboard on the UI and the BE only performs the mv/cp, perhaps the end points belong rather to `/content` and there, `copy` and `move` would be clearer.

Reply:
@rart yes, the UI will need to hold the items and know if the operation to perform is a copy or cut, a single call will be made to the backend